rfcs/2021-11-22-design-tokens.md

```diff
@@ -0,0 +1,108 @@
+
+Design tokens can be packaged from the atomic level to the application level by utilizing them in a dependency tree where one value inherits a value from the trunk above it.
+
+A design token solution is needed in which:
+
+1. The set of design tokens remain finite and equal
+2. The set of design tokens maintain consistency between application (DSM, Sketch, React and other future platforms such as iOS, Android, Web Components)
+3. The set of design tokens can be transposed, updated and extended easily and quickly across all systems at once
+4. The set of design tokens can be versioned and changes in that system can be tracked and monitored
+
+By deciding on a [Single Source Of Truth](https://en.wikipedia.org/wiki/Single_source_of_truth) (SSOT) for our design rules, we can organize and transform them for any system which utilizes them.
+
+## Benefits
+
+1. **Consistency** - Between design and code, utilizing a single set of variables everywhere allows a design proof to accurately describe the visual representation of a component as a set of variables. This provides better parity between what's designed and what is seen in the rendered usage on all platforms.
+
+   Looking at the [CSSStats for healthcare.gov](https://cssstats.com/stats?url=healthcare.gov), there are currently 70 color variations, many of which are nearly identical. Utilizing design tokens could bring this number down considerably and provide better brand consistency.
+
+2. **Organization** - By starting from the atomic level, we can use naming conventions for our tokens that are simple and specific to the variable they represent. By keeping these tokens separate from the systems they are utilized in, they can be managed as a discrete unit. Agreement on naming conventions of variables could eventually create parity between Sketch libraries and DS SASS variables.
+3. **Interoperability** - Design tokens allow a huge amount of flexibility for moving a design system to a new platform. Should an iOS/Android app ever be built, a new web application created, etc.. That system would have a framework ready to utilize for its theming which would be consistent with the other brand applications.
+4. **Opportunity for theming** - Decoupling design rules from the codebase allowing the transposition-ing of these static design decisions in various ways, i.e. theming/high contrast. By replacing all tokens in a system with new tokens you can essentially theme swap, assuming the consuming system is utilizing tokens at all levels appropriately.
```

Comment:
Yes it would be easier to evaluate what tokens are being used and which are not with tokens assuming the system utilizes them in some way everywhere.  The inheritance should always lead back to a top level token somewhere.